spec/serializers/evss_claim_detail_serializer_spec.rb

```diff
@@ -33,6 +33,15 @@
     end
   end
 
+  context 'with HTML in the VA representative field' do
+    let(:evss_claim) do
+      FactoryBot.build(:evss_claim, data: {'poa': '&lt;VATreatmentCenter&gt;'})
+    end
+    it 'strips the HTML tags' do
+      expect(attributes['va_representative']).to eq('VATreatmentCenter')
```

Comment:
`evss_claim = build(:evss_claim, data: {'poa': '&lt;VATreatmentCenter&gt;'})`

Everything about the spec all in one place

Reply:
This actually would fail, we'd need to change each instance of the `let(:evss_claim ...)` to use a locally scoped variable. Also `let ....` is canonical, and has benefits like memoization if it's used across classes. 

Additional point against moving to local var is that it's not immediately clear why we're not using the assigned var in that expect block unless you scroll up and read the initial `let()` and `subject()` declarations